app/src/main/java/com/framgia/music_44/screen/home/adapter/HomeAdapter.java

```diff
@@ -0,0 +1,82 @@
+package com.framgia.music_44.screen.home.adapter;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+import com.bumptech.glide.Glide;
+import com.framgia.music_44.R;
+import com.framgia.music_44.data.model.Songs;
+import com.framgia.music_44.util.OnItemClickListener;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.framgia.music_44.util.Constant.ONE;
+
+public class HomeAdapter extends RecyclerView.Adapter<HomeAdapter.ItemViewHolder> {
+    private List<Songs> mSongs;
+    private Context mContext;
+    private OnItemClickListener mListener;
+
+    public HomeAdapter(Context context, OnItemClickListener listener) {
+        mContext = context;
+        mListener = listener;
+        mSongs = new ArrayList<>();
+    }
+
+    public void addSongs(List<Songs> songs) {
+        mSongs.addAll(songs);
+        notifyItemRangeChanged( mSongs.size() - ONE,songs.size() - ONE);
+    }
+
+    @NonNull
+    @Override
+    public ItemViewHolder onCreateViewHolder(@NonNull ViewGroup viewGroup, int i) {
+        LayoutInflater layoutInflater = LayoutInflater.from(viewGroup.getContext());
+        View view = layoutInflater.inflate(R.layout.item_list, viewGroup, false);
+        return new ItemViewHolder(view, mListener);
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ItemViewHolder ItemViewHolder, int i) {
```

Comment:
check convention name
```suggestion
     public void onBindViewHolder(@NonNull ItemViewHolder itemViewHolder, int i) {
```